metatron-app/metatron-discovery
# #2625 Improves the Layers menu in the panel for dimension filters.

### Description
In the Dimension Filter panel, add layer menus for selection type and sorting.

**Related Issue** : #2625 

### How Has This Been Tested?
1. Create a dashboard.
2. Enter the edit screen and add one dimension filter.
3. In the Filter panel, expand the Layers menu and verify that the Sort / Selection Type menu is available.
4. Confirm that the list changes according to the menu selection.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
